usaspending_api/settings.py

```diff
@@ -48,7 +48,7 @@
     USASPENDING_AWS_REGION = os.environ.get("USASPENDING_AWS_REGION")
 
 # AWS locations for CSV files
-CSV_LOCAL_PATH = str(BASE_DIR / "csv_downloads")
+CSV_LOCAL_PATH = str(BASE_DIR / "csv_downloads") + "/"
 DOWNLOAD_ENV = ""
 BULK_DOWNLOAD_LOCAL_PATH = str(BASE_DIR / "bulk_downloads")
```

Comment:
and this line?

Reply:
I don't see this setting actually used for anything. Might be wise to update it anyway